test/integration/homepage_popular_links_test.rb

```diff
@@ -0,0 +1,35 @@
+require "integration_test_helper"
+
+class HomepagePopularLinksTest < JavascriptIntegrationTest
+  setup do
+    setup_users
+    @popular_links = FactoryBot.create(:popular_links)
+    visit_popular_links
+  end
+
+  should "show page title" do
+    assert_title "Popular on GOV.UK"
+  end
+
+  should "show homepage as a page title context" do
```

Comment:
Is "as" meant to be "has" here? Otherwise, this test name doesn't quite make sense to me.